MapboxNavigationUI/Resources/Navigation.storyboard

```diff
@@ -0,0 +1,682 @@
+                                        <rect key="frame" x="124" y="166" width="134" height="44"/>
+                                        <color key="backgroundColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
+                                        <constraints>
+                                            <constraint firstAttribute="height" constant="44" id="D6k-mC-uye"/>
+                                            <constraint firstAttribute="width" constant="134" id="S8H-gY-axx"/>
+                                        </constraints>
+                                        <fontDescription key="fontDescription" type="boldSystem" pointSize="16"/>
+                                        <inset key="contentEdgeInsets" minX="-8" minY="0.0" maxX="0.0" maxY="0.0"/>
+                                        <inset key="titleEdgeInsets" minX="16" minY="0.0" maxX="0.0" maxY="0.0"/>
+                                        <state key="normal" title="Recenter" image="recenter"/>
+                                        <userDefinedRuntimeAttributes>
+                                            <userDefinedRuntimeAttribute type="boolean" keyPath="whiteButton" value="YES"/>
+                                            <userDefinedRuntimeAttribute type="boolean" keyPath="hasBorder" value="NO"/>
+                                            <userDefinedRuntimeAttribute type="boolean" keyPath="hasShadow" value="YES"/>
+                                        </userDefinedRuntimeAttributes>
+                                        <connections>
+                                            <action selector="recenter:" destination="3z2-H5-unA" eventType="touchUpInside" id="Qzd-Ij-fUb"/>
+                                        </connections>
+                                    </button>
+                                </subviews>
+                                <color key="backgroundColor" red="0.90196078430000004" green="0.89019607840000003" blue="0.87450980389999999" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                                <constraints>
+                                    <constraint firstItem="yQ4-ig-aIN" firstAttribute="centerX" secondItem="nNr-30-cGD" secondAttribute="centerX" constant="3.5" id="7jk-0e-px2"/>
+                                    <constraint firstAttribute="bottom" secondItem="yQ4-ig-aIN" secondAttribute="bottom" constant="114" id="Ocm-nV-pGj"/>
+                                </constraints>
+                                <userDefinedRuntimeAttributes>
+                                    <userDefinedRuntimeAttribute type="boolean" keyPath="showsUserLocation" value="YES"/>
```

Comment:
The example applications’ storyboards set the style URL to Streets v10, as opposed to Streets v9, which is the default (but see mapbox/mapbox-gl-native#6301). If we use Streets for both, we should pick the same version for both.